README.md

```diff
@@ -1 +1,84 @@
-# cobol-to-avro-transform
+Cobol to Avro Converter
+===================================
+
+<a href="https://cdap-users.herokuapp.com/"><img alt="Join CDAP community" src="https://cdap-users.herokuapp.com/badge.svg?t=cobol-to-avro-transform"/></a>
+[![Build Status](https://travis-ci.org/hydrator/cobol-to-avro-transform.svg?branch=develop)](https://travis-ci.org/hydrator/cobol-to-avro-transform) [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) <img src="https://cdap-users.herokuapp.com/assets/cdap-action.svg"/>
+
+Cobol to Avro Converter reads the Cobol copybook and converts it into the Avro schema. Generated Avro schema
+is used to convert the records in the Cobol data file into Apache Avro format.
+
+Usage Notes
+-----------
+
+The plugin is used to convert the Cobol Copybook into Avro schema and use that schema to convert Cobol records into the Avro format.
```

Comment:
The plugin accepts a Cobol Copybook as a configuration to generate an avro schema and use that schema to convert records into avro format.